src/main/java/org/fundacion/jala/converter/controller/ProjectController.java

```diff
@@ -50,22 +59,71 @@ public int createProject(final @RequestParam("projectName") String projectName,
     /**
      * Endpoint for creating a project in data base.
      *
-     * @param fileName is a String with the file's name.
+     * @param fileName  is a String with the file's name.
      * @param idProject is an integer with the project's id.
      * @param extension is a String with file's extension.
-     * @param code is a String with the code for the file.
+     * @param code      is a String with the code for the file.
      * @return String with the path of files.
      * @throws IllegalStateException when method invoked at an illegal time.
-     * @throws IOException is a exception when invalid input is provided.
+     * @throws IOException           is a exception when invalid input is provided.
      */
     @PostMapping("/projects/{id}/file")
     public String createFiles(final @RequestParam("fileName") String fileName,  final @PathVariable("id") int idProject,
                               final @RequestParam("extension") String extension,
                               final @RequestParam("code") String code) throws IllegalStateException, IOException {
         Project project = ProjectSQL.findProjectById(idProject);
         String pathFile = project.getPath();
-        FileSQL.insertFileData(fileName, pathFile, idProject);
+        FileSQL.insertFileData(fileName + "." + extension, pathFile, idProject);
         Transform.createFile(code, fileName, extension, pathFile);
         return pathFile;
     }
+
+    /**
+     * Endpoint for creating a project in data base.
+     *
+     * @param idFile is a String with the file's name.
+     * @throws IllegalStateException when method invoked at an illegal time.
+     */
+    @DeleteMapping("/projects/file/{id}")
+    public void deleteFile(final @PathVariable("id") int idFile) throws IllegalStateException {
+        File file = FileSQL.findFileById(idFile);
+        String command = "del " + file.getPathFile() + "\\" + file.getName();
+        RunCommand runCommand = new RunCommand();
+        runCommand.run(command);
+        FileSQL.deleteFile(idFile);
+    }
+
+    /**
+     * Endpoint for creating a project in data base.
+     *
+     * @param idProject is an integer with the project's id.
+     * @return String with the path of files.
+     * @throws IllegalStateException when method invoked at an illegal time.
```

Comment:
instead of "method invoked", it should be "method is invoked"

Reply:
It is done.